src/style-value.js

```diff
@@ -16,15 +16,24 @@
 
   function StyleValue() {}
 
-  // TODO: Include parsing logic here.
   StyleValue.parse = function(property, value) {
-    if (typeof value == 'string') {
-      numberValue = Number.parseFloat(value);
-      if (numberValue !== NaN) {
-        return new scope.NumberValue(numberValue);
+    if (!propertyDictionary().isSupportedProperty(property)) {
+      return null;
+    }
+
+    if (propertyDictionary().isValidKeyword(property, value)) {
+      return Keyword.parse(value);
```

Comment:
Isn't this just the same as `new KeywordValue(value)`? It's clearer just to use the constructor in this case.